sw/device/tests/clkmgr_sleep_frequency_test.c

```diff
@@ -142,8 +142,10 @@ bool test_main(void) {
   CHECK(isr_entered);
 
   // Interrupt happened. Check the measurement state.
-  CHECK(clkmgr_testutils_check_measurement_counts(&clkmgr));
-  CHECK(clkmgr_testutils_check_measurement_enables(&clkmgr, kDifToggleEnabled));
+  CHECK_STATUS_OK(clkmgr_testutils_check_measurement_counts(&clkmgr));
+  bool all_enabled = CHECK_STATUS_OK(
+      clkmgr_testutils_check_measurement_enables(&clkmgr, kDifToggleDisabled));
```

Comment:
same here: why is a `CHECK_STATUS_OK` being assigned to a boolean?

Reply:
I created a `UNWRAP` macro.